src/main/kotlin/lotto/controller/LottoGame.kt

```diff
@@ -1,34 +1,62 @@
 package lotto.controller
 
+import lotto.model.AutomaticLottoTicketGenerator
 import lotto.model.LottoNumber
 import lotto.model.LottoTicket
-import lotto.model.RandomLottoTicketGenerator
+import lotto.model.ManualLottoTicketGenerator
 import lotto.model.TicketQuantity
 import lotto.model.WinningCalculator
 import lotto.model.WinningStatistics
 import lotto.view.InputView
-import lotto.view.ResultView
+import lotto.view.OutputView
 
 class LottoGame {
-    private val resultView = ResultView()
+    private val outputView = OutputView()
     private val inputView = InputView()
 
     fun start() {
-        val ticketQuantity = TicketQuantity(InputView().getAmountOfMoney()).quantity
-        resultView.showQuantity(ticketQuantity)
-        val tickets = purchaseLottoTicket(ticketQuantity)
-        resultView.showLottoTicket(tickets)
-        val lottoTicket = LottoTicket(inputView.getWinnerNumber())
+        val automaticTicketQuantity = TicketQuantity(inputView.getAmountOfMoney()).quantity
+        val manualTicketQuantity = inputView.getManualQuantity().toInt()
+        val manualLottoTickets =
+            ManualLottoTicketGenerator(purchaseManualLottoTicket(manualTicketQuantity)).tickets
+        outputView.showQuantity(manualTicketQuantity, automaticTicketQuantity)
```

Comment:
manualLottoTickets와 무관하게 quantity만을 이용해 showQuantity를 수행할 수 있으므로 두 순서를 바꾸어도 좋을 것 같네요.

```suggestion
        outputView.showQuantity(manualTicketQuantity, automaticTicketQuantity)
        val manualLottoTickets = ManualLottoTicketGenerator(purchaseManualLottoTicket(manualTicketQuantity)).tickets
```

Reply:
컨트롤러 부분을 전체적으로 리팩터링 해보겠습니다.